src/main/resources/templates/fragments/footer.html

```diff
@@ -69,7 +69,7 @@ <h6 class="text-uppercase font-weight-bold">
     <!--/.Footer Links-->
 
     <!-- Copyright -->
-    <div class="footer-copyright text-center py-3">© 2020 Copyright:
+    <div class="footer-copyright text-center py-3">© <script>document.write(new Date().getFullYear())</script> Copyright:
```

Comment:
Preferowane byłoby rozwiązanie tego z użyciem Thymeleafa, a nie JavaScriptu. Po wyłączeniu javascriptu w przeglądarce to też powinno działać